wordpress-20-04/template.json

```diff
@@ -0,0 +1,86 @@
+
+{
+  "variables": {
+    "do_api_token": "{{env `DIGITALOCEAN_API_TOKEN`}}",
+    "image_name": "wordpress-20-04-snapshot-{{timestamp}}",
+    "apt_packages": "apache2 fail2ban libapache2-mod-php mysql-server php php-apcu php-bz2 php-curl php-gd php-gmp php-intl php-json php-mbstring php-mysql php-pspell php-soap php-tidy php-xml php-xmlrpc php-xsl php-zip postfix python3-certbot-apache software-properties-common unzip",
+    "appiication_name": "WordPress",
+    "appiication_version": "5.5.1",
```

Comment:
We are going to explicitly select which version of WordPress we want to install. We should automate triggering a new build anytime a new WordPress version is released.